edgelet/edgelet-http-mgmt/src/server/module/create.rs

```diff
@@ -50,38 +54,63 @@ where
                 })
                 .and_then(move |(spec, core_spec)| {
                     let module_name = spec.name().to_string();
-                    runtime
-                        .registry()
-                        .pull(core_spec.config())
-                        .then(move |result| match result {
-                            Ok(_) => Ok(runtime.create(core_spec).then(
-                                move |result| -> Result<_, Error> {
+                    let pull_policy = *core_spec.pull_policy();
```

Comment:
You don't need the `*` since match ergonomics will automatically change the match arm patterns to refs.

Reply:
Same reason as in prepare_update.rs